### DISCUSSION TITLE
一言API是不是炸了，今天的内容有点辣眼睛

### DISCUSSION THREAD

**Discussion by HanHan666666** [solstice23/argon-theme#559] [Q&A]
![image](https://user-images.githubusercontent.com/60591430/198470198-aa6bb1a4-e0c7-4cd4-b185-04abb3dae83d.png)
![image](https://user-images.githubusercontent.com/60591430/198470339-dab833e6-4e07-44f2-aac1-a84d92302bbc.png)

**Comment by steveji2021** [comment]
哈哈哈哈

**Comment by God-2077** [comment]
要么[一言](https://github.com/solstice23/argon-theme/discussions/559#top)被黑了，要么就是没审查给放上[一言](https://github.com/solstice23/argon-theme/discussions/559#top)了
